src/pages/HammingCode.vue

```diff
@@ -60,17 +58,20 @@ export default defineComponent({
     const { urlRef } = useUrlRef(router, route);
 
     const code = urlRef("code", "");
-    // TODO: make urlRef support booleans
-    const dataOnly = urlRef("dataOnly", "false");
+    const dataOnly = urlRef("dataOnly", false);
+
+    // url cleanup
+    code.value = code.value.replace(/[^01]/g, '');
+    console.log(dataOnly.value);
```

Comment:
Let's remove the console.log before merging this

Reply:
done :)